src/30_Advanced/ComboBox.html

```diff
@@ -0,0 +1,181 @@
+  
+  <style amp-custom>
+        .hidden, .hidden.select-option {
+            display: none;
+        }
+
+        .arrow-up {
+            transform: rotate(180deg);
+        }
+
+        .product-search{
+            width: 100%;
+            max-width: 350px;
+        }
+
+        .search-container {
+            display: flex;
+            flex-flow: row nowrap;
+            position: relative;
+            margin-top: 20px;
+        }
+
+        .search-submit {
+            font-size: 1.25em;
+            padding-right: 13px;
+        }
+
+        .autosuggest-box {
+            box-shadow: 0px 2px 6px rgba(0,0,0,.3);
+            overflow-y: scroll;
+        }
+
+        input.search-box {
+            padding: 5px 8px;
+            font-size: 1em;
+            background-color: #fff;
+            font-family: Arial,"Helvetica Neue",Helvetica,sans-serif;
+            border: 1px solid #d4d4d4;
+            line-height: 1.53333em;
+            height: 2em;
+            padding-right: 50px;
+            width: 100%;
+                   
+        }
+
+        .select-option {
+            display: block;
+            box-sizing: border-box;
+            height: 30px;
+            line-height: 30px;
+            padding-left: 10px;
+            font-size: 15px;
+        }
+
+        amp-selector .select-option.no-outline.no-outline[option] {
+            outline: none;
+        }
+
+        .select-option:focus {
+            box-shadow: inset rgb(94, 158, 215) 0px 0px 45px;
+        }
+
+        .select-option:focus {
+            background: #fff;
+        }
+
+
+        .lvl3 {
+            padding-left: 10px;
+            color: #777
+        }
+
+        .lvl2 {
+            padding-left: 5px;
+            font-weight: bold;
+            color: #777
+        }
+
+        .lvl1 {
+            padding-left: 0px;
+            color: #ccc;
+            font-weight: bold;
+        }
+
+        .select-option-wrap {
+            padding: 0 20px;
+                    
+        }
+
+        .dropdown-wrap {
+            padding-top: 2em;
+            padding-left: 1em;
+        }
+
+        .dropdown-selector {
+            max-height: 420px;
+            overflow-y: scroll;
+        }
+    </style>
+</head>
+<body>
+
+  <!-- ## Basic Usage -->
+  <!--
+  This example includes two inputs. One is the input in which the user writes the text and the other is a hidden input, 
+  where the id value of the selected option is stored and could be submited in a form.
+
+  The dropdown list's options are rendered server side, instead of using amp-list to maximize responsiveness. 
+  The 'option' attribute of the amp-selector's options, contain the id of the element, as well as its title, seperated by a comma.
```

Comment:
s/'option'/`option`

nice trick with storing title and id in the option 👍

Reply:
Done!